cmd/registry/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	_ "github.com/docker/distribution/registry/storage/driver/filesystem"
 	_ "github.com/docker/distribution/registry/storage/driver/inmemory"
 	_ "github.com/docker/distribution/registry/storage/driver/middleware/cloudfront"
+	_ "github.com/docker/distribution/registry/storage/driver/rados"
```

Comment:
Ok, I'll add it tomorrow morning!

On Fri, May 15, 2015, 03:07 Stephen Day <notifications@github.com> wrote:

> In cmd/registry/main.go
> <https://github.com/docker/distribution/pull/443#discussion_r30378573>:
>
> > @@ -25,6 +25,7 @@ import (
> >  	_ "github.com/docker/distribution/registry/storage/driver/filesystem"
> >  	_ "github.com/docker/distribution/registry/storage/driver/inmemory"
> >  	_ "github.com/docker/distribution/registry/storage/driver/middleware/cloudfront"
> > +	_ "github.com/docker/distribution/registry/storage/driver/rados"
>
> Just have a file in the main package, rados.go with following in it:
>
> // +build include_rados
> package main
> import _ "github.com/docker/distribution/registry/storage/driver/rados"
>
> I'd rather not have an empty package laying around, if at all possible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/distribution/pull/443/files#r30378573>.
>